### ISSUE TITLE
enchantment for pickaxe

### ISSUE THREAD

**Issue by ShyamKa** [pronze/SBAHypixelify#39]
i want to have enchant
permeant and not get gold pick
which ever pickaxe i get
like sharpen swords

**Comment by SamSamsung** [comment]
Yeah, i would like to have that to, that's a litlle annoying. I would really appreciate if you make that feature !

**Comment by Square661** [comment]
> i want to have enchant
> permeant and not get gold pick
> which ever pickaxe i get
> like sharpen swords

Ever think that you could be a bit nicer when asking for something?

**Comment by SamSamsung** [comment]
I do agree with you
